bedrock/base/templates/base-error.html

```diff
@@ -12,8 +12,10 @@
 {% block site_header %}
   <header class="mzp-l-content header-spacing">
     <div class="header-image">
-    {{ high_res_img('img/co-brand/firefox_moz_co-brand_sm.png', {'alt': 'Firefox, Mozilla'}) }}
+    <a href="https://www.firefox.com">{{ high_res_img('img/logos/firefox/logo-reality.png', {'alt': 'Firefox'}) }}</a>
     </div>
+    <div class="header-image">
+      <a href="https://www.mozilla.org"> {{ high_res_img('img/logos/mozilla/black.svg', {'alt': ' Mozilla'}) }}</a>
```

Comment:
There's no need to use `high_res_img()` helper for an svg asset.

I'd suggest using `protocol/img/logos/mozilla/black.svg`

The `href` should also use the URL helper `href="{{ url('mozorg.home') }}"`